upgrades.yaml

```diff
@@ -90,3 +90,6 @@ operations:
   - version: 4.4.0
     pre:
       - upgrades/pre/upgrade-4-4-0.mjs || true
+  - version: 4.4.5
```

Comment:
I think that we should update the version number of the values. 
and migration should happen during the migrate phase.